src/tests/ftest/pool/attribute.py

```diff
@@ -100,7 +103,7 @@ def verify_get_attr(self, indata, outdata):
         """
         self.log.info("Verifying get_attr output:")
         self.log.info("  get_attr data: %s", indata)
-        self.log.info("  set_attr date: %s", outdata)
+        self.log.info("  set_attr data: %s", outdata)
```

Comment:
This looks backwards - the `indata` should be what was used when calling `set_attr` and `outdata` should be what was returned from calling `get_attr` (see `Args` above):
```
        self.log.info("  set_attr data: %s", indata)
        self.log.info("  get_attr data: %s", outdata)
```

Reply:
I can change the names of the arguments, the test use different names when calling the function compared to the arguments name in the function, no problem